packages/loot-core/src/client/shared-listeners.ts

```diff
@@ -119,15 +128,16 @@ export function listenForSyncEvent(actions, store) {
           // Tell the user something is wrong with the key state on
           // the server and the key needs to be recreated
           notif = {
-            title: 'Your encryption key need to be reset',
+            title: i18n.t('Your encryption key need to be reset', {}),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
            title: i18n.t('Your encryption key need to be reset'),
```